web/.env.example

```
@@ -0,0 +1,4 @@
+DATABASE_URL=postgres://career_ops:career_ops_dev@localhost:5434/career_ops
+PORT=3000
+# Absolute path to the career-ops project root (parent of web/)
+CAREER_OPS_ROOT=..
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CAREER_OPS_ROOT key should go before the DATABASE_URL key

(UnorderedKey)